src/testFixtures/java/uk/gov/di/ipv/cri/common/library/stepdefinitions/CommonSteps.java

```diff
@@ -123,6 +126,15 @@ public void userSendsAPostRequestToTokenEndpoint() throws IOException, Interrupt
         this.testContext.setResponse(this.commonApiClient.sendTokenRequest(privateKeyJWT));
     }
 
+    @When("user sends a GET request to events end point")
+    public void userSendsAGetRequestToEventsEndpoint() throws IOException, InterruptedException {
+        this.testContext.setResponse(
+                this.testResourcesClient.sendEventRequest(this.testContext.getSessionId()));
+
```

Comment:
remove this please

Reply:
This is needed to reference the address feature file

Reply:
Confirm - this is needed